recipes/r-reldist/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set version = '1.6-6' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-reldist
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/reldist_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/reldist/reldist_{{ version }}.tar.gz
+  sha256: 8a7b55120d24fae72b4c284072b07d7f90094ba00bf89ba172e0b3ab8fbe7127
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-hmisc
+    - r-mgcv
+  run:
+    - r-base
+    - r-hmisc
+    - r-mgcv
+
+test:
+  commands:
+    - $R -e "library('reldist')"           # [not win]
+    - "\"%R%\" -e \"library('reldist')\""  # [win]
+
+about:
+  home: http://www.stat.ucla.edu/~handcock/RelDist
+  license: GPL-3
+  summary: Tools for the comparison of distributions. This includes nonparametric estimation
+    of the relative distribution PDF and CDF and numerical summaries as described in
+    "Relative Distribution Methods in the Social Sciences" by Mark S. Handcock and Martina
+    Morris, Springer-Verlag, 1999, Springer-Verlag, ISBN 0387987789.
+  license_family: GPL3
```

Comment:
why is the generator not adding `license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'`

Reply:
It probably doesn't exist. Last I looked CRAN didn't actually enforce this (maybe it does for new packages).

Reply:
Hmm, in this case it's just `LICENSE`. I'll try to manually add these in.

Reply:
ah. `'{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'` is referring to a file provided r-base package, not by the cran tarball of reldist.

Reply:
The skeleton creator normally add those lines by matching the cran meta data... see https://github.com/conda/conda-build/blob/master/conda_build/skeletons/cran.py#L1430 till line 1441 ... sometimes this seems not to work.